src/transaction.js

```diff
@@ -62,6 +63,11 @@ function wrapper( client, parent_logger ){
 
           if( task.status > 201 ){
             logger.error( '[' + action.status + ']', action.error );
+
+            // additional debugging output of the error and corresponding request
+            var req = batch._slots.filter(s => _.get(s, 'cmd.index._id') === action._id);
+            logger.info(JSON.stringify(action, null, 2));
```

Comment:
what about the `warn` level? That implies the output is related to a problem much more than `info`.